python/src/MartinezSensitivityAlgorithm_doc.i.in

```diff
@@ -52,6 +52,7 @@ sample has a zero mean) defined by:
    \frac{\sum_{k=1}^N z_k z_k'}
    {\sqrt{\sum_{k=1}^N z_k^2} \sqrt{\sum_{k=1}^N \left( z_k' \right)^2}}
 
+for any pair of vectors :math:`\vect{z}, \vect{z}' \in \Rset^N`
```

Comment:
IMO this line you added should be moved up to line 47:

"... has a zero mean) defined for any pair of vectors :math:`\vect{z}, \vect{z}' \in \Rset^N` by: "

Reply:
Ok : fixed at 1b5ad3206